### DISCUSSION TITLE
Arduino IDE 2.0 has moved to stable and is available for download

### DISCUSSION THREAD

**Discussion by bwims** [earlephilhower/arduino-pico#895] [Q&A] [answered]
Hi,

IDE 2.0 has moved out of beta. Is it ok to upgrade?

[Arduino IDE 2.0 has moved to stable and is available for download](https://blog.arduino.cc/2022/09/14/its-here-please-welcome-arduino-ide-2-0/)

Thanks!

PS wasn't sure if this was an issue or discussion item!

**Comment by nirbkm** [comment, marked as answer]
Hi, 
I installed the library on the last version of IDE 2.0 and did some basic usage, for now working without any issues,
also couldn't find any formal issues reported.

**Reply by bwims** [reply]
Many thanks for that!
